Makefile

```diff
@@ -242,6 +242,11 @@ ifeq ($(USE_NWAKU), true)
     NIM_EXTRA_PARAMS += --passL:"-L$(LIBWAKU_LIBDIR)" --passL:"-lwaku"
 endif
 
+LIBSDS_LIBDIR := $(GIT_ROOT)/vendor/nim-sds/build
+NIM_EXTRA_PARAMS += --passL:"-L$(LIBSDS_LIBDIR)" --passL:"-lsds"
+STATUSGO_MAKE_PARAMS += NIM_SDS_HEADER_PATH="$(LIBSDS_LIBDIR)"
+STATUSGO_MAKE_PARAMS += NIM_SDS_LIB_PATH="$(GIT_ROOT)/vendor/nim-sds/build/"
+
```

Comment:
Look how nwaku is integrated here:
https://github.com/status-im/status-desktop/blob/7c867c40dc656b9da3abac43fde86eab2c6b56c9/Makefile#L238-L243

We should use the `NIM_SDS_SOURCE_PATH` in this case, not header/lib.